README.md

```diff
@@ -12,39 +12,40 @@ The documentation is available at [trento-project.io/web](https://trento-project
 
 Swagger UI is available at [trento-project.io/web/swaggerui](https://trento-project.io/web/swaggerui).
 
-# Overview
+# Overview of Trento
 
-_Trento_ is a comprehensive cloud-native, distributed monitoring solution.
+**Trento** is a comprehensive cloud-native, distributed monitoring solution, and it consists of three main components:
```

Comment:
ocd: Something we have **Trento** in bold, others in _Trento_ , and other without any